README.md

```diff
@@ -29,34 +29,28 @@
 
     We now have an AKS cluster named `bikesharingcluster` in resource group `bikesharinggroup` in the `eastus2` region with Azure Dev Spaces enabled.
 
-4. Select the `dev` parent space:
-    
-    ```
-    azds space select -n dev
-    ```
-
-5. We now need to retrieve the host suffix. To do this, run the following command and note the `HostSuffix` field's value:
+4. We now need to retrieve the host suffix. To do this, run the following command and note the `HostSuffix` field's value:
 
     ```
     azds show-context
     ```
 
-6. We have to update some of our code with this host suffix. Open the `./chart/values.yaml` file and replace the occurrences of `<REPLACE_ME_WITH_HOST_SUFFIX>` in this file with the host suffix.
+5. We have to update some of our code with this host suffix. Open the `./chart/values.yaml` file and replace the occurrences of `<REPLACE_ME_WITH_HOST_SUFFIX>` in this file with the host suffix.
 
-7. From the source repository's root folder, navigate to the `chart` folder:
+6. From the source repository's root folder, navigate to the `chart` folder:
```

Comment:
>6 [](start = 0, length = 1)

nit: To avoid having to update all the number values, you should be able to use `1.` for each and most Markdown renders will properly count up for you.

Reply:
As long as any sub-content is indented, to indicate a continuation of the numbered list.

---
In reply to: [268899845](https://github.com/ContosoBikeRental/BikeSharingSampleApp/pull/26#discussion_r268899845) [](ancestors = 268899845)

Reply:
Thanks for the tip! Let's update this list. :)

---
In reply to: [268899959](https://github.com/ContosoBikeRental/BikeSharingSampleApp/pull/26#discussion_r268899959) [](ancestors = 268899959,268899845)